whatlies/embeddingset.py

```diff
@@ -95,6 +95,28 @@ def __contains__(self, item):
         """
         return item in self.embeddings.keys()
 
+    def __getitem__(self, thing):
```

Comment:
Why the change in position of this method?!

Reply:
All the dunder methods are now together. I allmost found myself re-implementing that method because it wasn't near the others can I didn't find it. With a similar philosophy all the plotting methods are together at the bottom of the object.

Reply:
Ah, I see. All right!